### PR TITLE
Add support info about position:[relative|sticky] for <table> elements

### PR DESCRIPTION
**Pull request by chrisdavidmills**

### CODE REVIEW

**Comment by flying-sheep** [comment]
I think chrome had this working for table elements at some point and removed it again.

**Comment by mkurz** [comment]
@flying-sheep Check https://bugs.chromium.org/p/chromium/issues/detail?id=702927

**Comment by flying-sheep** [comment]
Thanks! I mentioned that because it could be included in the compat table.

**Comment by chrisdavidmills** [comment]
Thanks @flying-sheep / @mkurz  — I've updated the support info for Chrome/Opera according to the chromium bug link info. *think* I've got it right.

File: css/properties/position.json [hunk 1]
```diff
@@ -210,6 +210,54 @@
               "deprecated": false
             }
           }
+        },
+        "position_sticky_table_elements": {
+          "__compat": {
+            "description": "Table elements as <code>sticky</code> positioning containers",
+            "support": {
+              "webview_android": {
+                "version_added": "64"
```
Comment (teoli2003) [inline css/properties/position.json]: According to the linked bug, this has been backed out. We should keep it at false, until it is indeed fixed.

File: css/properties/position.json [hunk 2]
```diff
@@ -210,6 +210,54 @@
...
+              },
+              "chrome": {
+                "version_added": "64"
```
Comment (teoli2003) [inline css/properties/position.json]: According to the linked bug, this has been backed out. We should keep it at false, until it is indeed fixed.

File: css/properties/position.json [hunk 3]
```diff
@@ -210,6 +210,54 @@
...
+              },
+              "chrome_android": {
+                "version_added": "64"
```
Comment (teoli2003) [inline css/properties/position.json]: According to the linked bug, this has been backed out. We should keep it at false, until it is indeed fixed.

File: css/properties/position.json [hunk 4]
```diff
@@ -210,6 +210,54 @@
...
+                "version_added": "59"
+              },
+              "firefox_android": {
+                "version_added": "59"
+              },
+              "ie": {
+                "version_added": null
+              },
+              "opera": {
+                "version_added": "51"
```
Comment (teoli2003) [inline css/properties/position.json]: According to the linked bug, this has been backed out. We should keep it at false, until it is indeed fixed.

File: css/properties/position.json [hunk 5]
```diff
@@ -210,6 +210,54 @@
...
+              },
+              "opera_android": {
+                "version_added": "51"
```
Comment (teoli2003) [inline css/properties/position.json]: According to the linked bug, this has been backed out. We should keep it at false, until it is indeed fixed.

**Comment by chrisdavidmills** [comment]
Changes made as requested.

**Pull request merged**